packages/manager/apps/pci-rancher/tsconfig.build.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "strict": false,
+    "lib": ["dom", "es2020", "ESNext"],
+    "noEmit": true,
+    "target": "ESNext",
+    "types": ["vite/client", "node"],
+    "module": "esnext",
+    "moduleResolution": "Node",
+    "removeComments": true,
+    "outDir": "dist",
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "noImplicitAny": true,
+    "declaration": true,
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "jsx": "react",
+    "paths": {
+      "@/*": ["./src/*"],
+      "@playwright-helpers/*": ["../../../../playwright-helpers/*"],
+      "@translation/*": ["./public/translations/*"]
+    }
+  },
```

Comment:
I think you should extend the base tsconfig instead to avoid setting each value again